src/components/Home/Home.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { useEffect, useState } from 'react';
+
+type Props = {};
+
+const Home = (props: Props): JSX.Element => {
+  const [city, setCity] = useState('London');
+  const [data, setData] = useState('');
+
+  useEffect(() => {
+    (async () => {
+      try {
+        const accessToken = localStorage.getItem('token');
+        const options = {
+          method: 'GET',
+          headers: {
+            Authorization: `Bearer ${accessToken}`
+          }
+        };
+        const response = await fetch(
+          `https://pfa.foreca.com/api/v1/location/search/${city}`,
```

Comment:
api url can be stored in separate variable to not duplicate it every time

Reply:
Yes, did that.